File: ProyectoMysql/Api/LogisticStorage/LogisticStorage.BusinessLayer/OrdenCompraDetalle.cs
using LogisticStorage.DataLayer;
using LogisticStorage.EntityLayer;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LogisticStorage.BusinessLayer
{
    public class OrdenCompraDetalle
    {
        public static List<OrdenCompraDetalleEntity> ObtenerItem(Int32 OrdenCompraId)
        {
            OrdenCompraDetalleDB DB = new OrdenCompraDetalleDB();
            return DB.ObtenerItem(OrdenCompraId);
        }
    }
}

File: ProyectoMysql/server/BusinessLayer/OrdenCompraDetalle.py
from DataLayer.OrdenCompraDetalleDB import *


class OrdenCompraDetalle:

    def ObtenerItem(Id: int)-> list[OrdenCompraSaveModel]:
        try:
            return OrdenCompraDetalleDB.ObtenerItem(Id)
        except Exception as e:
            print(e)

